docs/tutorial-script-single.rst

```diff
@@ -0,0 +1,280 @@
+.. _script_single:
+
+Creating the Azure environment for the "single standby" tutorial
+================================================================
+
+In the "single standby" tutorial we create a primary and secondary Postgres
+node and set up pg_auto_failover to replicate data between them. Then we
+simulate failure in the primary node and see how the system smoothly
+switches (fails over) to the secondary.
+
+For illustration, we run our databases on virtual machines in the Azure
+platform. The techniques here are relevant to any cloud provider or
+on-premise network. We use four virtual machines: a primary database, a
+secondary database, a monitor, and an "application." The monitor watches the
+other nodes’ health, manages global state, and assigns nodes their roles.
+
+This document shows you how to create the Azure environment with the virtual
+machines needed for the rest of the "single standby" tutorial. You have two
+options:
+
+  1. use a pg_autoctl command that automates all the steps so that you can
```

Comment:
Although this list explains that the `pg_autoctl do azure …` and `az … create` methods are two alternatives, the structure of the article makes it look like the reader should run both of them, one after another.

What if we split the two methods into their own pages, and linked to them in this list?